akka-cluster-sharding/src/test/scala/akka/cluster/sharding/CoordinatedShutdownShardingSpec.scala

```diff
@@ -76,13 +76,13 @@ class CoordinatedShutdownShardingSpec extends AkkaSpec(CoordinatedShutdownShardi
     awaitAssert({
       val p1 = TestProbe()(sys2)
       region2.tell(1, p1.ref)
-      p1.expectMsg(1.seconds, 1)
+      p1.expectMsg(1)
       val p2 = TestProbe()(sys2)
       region2.tell(2, p2.ref)
-      p2.expectMsg(1.seconds, 2)
+      p2.expectMsg(2)
       val p3 = TestProbe()(sys2)
       region2.tell(3, p3.ref)
-      p3.expectMsg(1.seconds, 3)
+      p3.expectMsg(3)
     }, 10.seconds)
```

Comment:
I could imagine that the reason for the shorter timeouts are because they are inside `awaitAssert` and if failing they will be retried more quickly. Perhaps increase the 10.seconds instead?